searchsummary/src/vespa/searchsummary/docsummary/docsumwriter.cpp

```diff
@@ -78,6 +78,9 @@ DynamicDocsumWriter::resolveInputClass(ResolveClassInfo &rci, uint32_t id) const
     }
 }
 
+constexpr uint32_t default_32bits_int = (uint32_t)std::numeric_limits<int32_t>::min();
```

Comment:
I suggest we use getUndefined() in searchcommon/src/vespa/searchcommon/common/undefinedvalues.h for this.

Reply:
done.